Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG DOCKER_REGISTRY=docker.io
-ARG TORCH_VERS=2.2.2
-ARG CUDA_VERS=12.1
+ARG TORCH_VERS
+ARG CUDA_VERS
```

Comment:
This will break our working runai since we don't pass these build-args (yet). why removing sound default values ?

Reply:
Hum, ok. That was to have a single reference of version in the `env.yaml` file, but ok I roll back